address;source_location;insn;indent
0x17bb80;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:94;PUSH	%RBP;
0x17bb81;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:94;MOV	%RSP,%RBP;
0x17bb84;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:94;PUSH	%R12;
0x17bb86;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:94;PUSH	%RBX;
0x17bb87;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:94;MOV	%RDI,%RBX;
0x17bb8a;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:95;CALL	106fe0 <PyObject_GC_UnTrack@plt>;
0x17bb8f;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;LEA	-0x16(%RIP),%RSI;
0x17bb96;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;MOV	%RBX,%RDI;
0x17bb99;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;CALL	1075c0 <_PyTrash_cond@plt>;
0x17bb9e;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;TEST	%EAX,%EAX;
0x17bba0;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;JNE	17bbc0 <BaseException_dealloc+0x40>;
0x17bba2;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:100;MOV	%RBX,%RDI;
0x17bba5;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:100;CALL	17b2e0 <BaseException_clear>;
0x17bbaa;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:101;MOV	0x8(%RBX),%RAX;
0x17bbae;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:101;MOV	%RBX,%RDI;
0x17bbb1;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;POP	%RBX;
0x17bbb2;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;POP	%R12;
0x17bbb4;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:101;MOV	0x140(%RAX),%RAX;
0x17bbbb;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;POP	%RBP;
0x17bbbc;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:101;JMP	%RAX;
0x17bbbe;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:101;XCHG	%AX,%AX;
0x17bbc0;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;CALL	10acf0 <_PyThreadState_UncheckedGet@plt>;
0x17bbc5;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;MOV	%RBX,%RSI;
0x17bbc8;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;MOV	%RAX,%RDI;
0x17bbcb;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;MOV	%RAX,%R12;
0x17bbce;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;CALL	108050 <_PyTrash_begin@plt>;
0x17bbd3;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;TEST	%EAX,%EAX;
0x17bbd5;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:99;JNE	17bc00 <BaseException_dealloc+0x80>;
0x17bbd7;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:100;MOV	%RBX,%RDI;
0x17bbda;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:100;CALL	17b2e0 <BaseException_clear>;
0x17bbdf;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:101;MOV	0x8(%RBX),%RAX;
0x17bbe3;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:101;MOV	%RBX,%RDI;
0x17bbe6;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:101;CALLQ	0x140(%RAX);
0x17bbec;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:102;TEST	%R12,%R12;
0x17bbef;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:102;JE	17bc00 <BaseException_dealloc+0x80>;
0x17bbf1;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;POP	%RBX;
0x17bbf2;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:102;MOV	%R12,%RDI;
0x17bbf5;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;POP	%R12;
0x17bbf7;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;POP	%RBP;
0x17bbf8;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:102;JMP	10b390 <_PyTrash_end@plt>;
0x17bbfd;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:102;NOPL	(%RAX);
0x17bc00;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;POP	%RBX;
0x17bc01;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;POP	%R12;
0x17bc03;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;POP	%RBP;
0x17bc04;/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/exceptions.c:103;RET;
0x17bc05;:0;NOPW	%CS:(%RAX,%RAX,1);
